src/DssSpell.t.sol

```diff
@@ -567,29 +567,25 @@ contract DssSpellTest is DssSpellTestBase {
-            Payee(wallets.addr("UPMAKER"),                        13.89 ether),  // Note: ether is a keyword helper, only MKR is transferred here
-            Payee(wallets.addr("VIGILANT"),                       13.89 ether),  // Note: ether is a keyword helper, only MKR is transferred here
-            Payee(wallets.addr("PBG"),                            13.44 ether),  // Note: ether is a keyword helper, only MKR is transferred here
-            Payee(wallets.addr("PIPKIN"),                         5.82 ether),  // Note: ether is a keyword helper, only MKR is transferred here
-            Payee(wallets.addr("QGOV"),                           4.48 ether),  // Note: ether is a keyword helper, only MKR is transferred here
-            Payee(wallets.addr("WBC"),                            4.03 ether),  // Note: ether is a keyword helper, only MKR is transferred here
-            Payee(wallets.addr("LAUNCH_PROJECT_FUNDING"),         500 ether),  // Note: ether is a keyword helper, only MKR is transferred here
-            Payee(wallets.addr("VENICE_TREE"),                    20.84 ether)  // Note: ether is a keyword helper, only MKR is transferred here
+        Payee[10] memory payees = [
+            Payee(wallets.addr("DEFENSOR"),    41.67 ether), // Note: ether is a keyword helper, only MKR is transferred here
+            Payee(wallets.addr("BLUE"),        41.67 ether), // Note: ether is a keyword helper, only MKR is transferred here
+            Payee(wallets.addr("BONAPUBLICA"), 41.67 ether), // Note: ether is a keyword helper, only MKR is transferred here
+            Payee(wallets.addr("CLOAKY"),      41.67 ether), // Note: ether is a keyword helper, only MKR is transferred here
+            Payee(wallets.addr("TRUENAME"),    41.67 ether), // Note: ether is a keyword helper, only MKR is transferred here
+            Payee(wallets.addr("PBG"),         13.89 ether), // Note: ether is a keyword helper, only MKR is transferred here
+            Payee(wallets.addr("UPMAKER"),     13.89 ether), // Note: ether is a keyword helper, only MKR is transferred here
+            Payee(wallets.addr("VIGILANT"),    13.89 ether), // Note: ether is a keyword helper, only MKR is transferred here
+            Payee(wallets.addr("WBC"),         13.89 ether), // Note: ether is a keyword helper, only MKR is transferred here
+            Payee(wallets.addr("JAG"),         13.71 ether)  // Note: ether is a keyword helper, only MKR is transferred here
         ];
         // Fill the value below with the value from exec doc
-        uint256 expectedSumPayments = 780.67 ether; // Note: ether is a keyword helper, only MKR is transferred here
+        uint256 expectedSumPayments = 277.62 ether; // Note: ether is a keyword helper, only MKR is transferred here
 
         // Calculate and save previous balances
         uint256 totalAmountToTransfer = 0; // Increment in the loop below
```

Comment:
Should we just revert this changes and only add the `skipped` modifier?

Reply:
I don't think it matters to be honest, since those are skipped anyway and there are no merge conflicts

Reply:
(but if it would be a regular spell, I would address this, just to preserve clean git history, of course)